helm/values.yaml

```diff
@@ -19,8 +19,9 @@ service:
         periodSeconds: 5
       readiness:
         initialDelaySeconds: 15
-        periodSeconds: 15
+        periodSeconds: 5
```

Comment:
can our `periodSeconds` be less than our `timeoutSeconds`?  Wouldn't that mean we could potentially have two, or more, readiness checks occurring in parallel?

Reply:
"you can say that they will accumulate if it means that you have more than 1 unresolved probe at the same time"
https://stackoverflow.com/questions/63308717/what-is-the-role-of-timeoutseconds-in-kubernetes-liveness-readiness-probes

Nice catch, I wasn't able to find people doing that and giving good reasons to do so. Do you have suggestions on numbers?

I'm thinking two possible options, either:
- Keep periodSeconds as 15 (Can take longer to recover, but it only needs one success check. For failing it needs 10 consecutive failures)
- Reduce/Remove timeoutSeconds (We won't be able to find a good number in our inconsistent scenario)

Reply:
How about we set `periodSeconds == timeoutSeconds` whereby `timeoutSeconds = 10`.  Theoretically, probes won't overlap as much in that setup.

Reply:
@blucas I'd still rather to have period a little higher than timeout. Even if it's just one second.

Reply:
As we have discussed, set period to 12s